### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ahmadali122**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1 @@
 # mslm.io
```
Comment (ahmadali122) [inline README.md:1]: jhfjhvgk

File: about.html [hunk 2]
```diff
@@ -12,17 +12,17 @@
...
 
     <body>
-        <div class="mob_navbar_bg"></div>
+        <!-- <div class="mob_navbar_bg"></div> -->
 
-        <div class="header_logo">
+        <!-- <div class="header_logo">
             <a href="./index.html"><img src="./images/Logo.svg" /></a>
-        </div>
+        </div> -->
```
Comment (Shoaib-Ashfaq) [inline about.html:20]: abcd

**Comment by Shoaib-Ashfaq** [comment]
bvxcgf

File: about.html [hunk 3]
```diff
@@ -12,17 +12,17 @@
     </head>
 
     <body>
-        <div class="mob_navbar_bg"></div>
+        <!-- <div class="mob_navbar_bg"></div> -->
```
Comment (Shoaib-Ashfaq) [inline about.html:15]: why thi

File: about.html [hunk 4]
```diff
@@ -12,17 +12,17 @@
...
-        </div>
+        </div> -->
 
         <div class="navbar">
             <img class="hamburger" src="./images/hamburger.svg" />
-            <a href="./index.html"
+            <!-- <a href="./index.html"
                 ><img class="navbar_logo" src="./images/Logo.svg"
-            /></a>
+            /></a> -->
```
Comment (Shoaib-Ashfaq) [inline about.html:25]: jbhxf

**Review by Shoaib-Ashfaq — CHANGES_REQUESTED** [review]
small cha

**Review by Shoaib-Ashfaq — APPROVED** [review]

**Pull request merged**